Prevent event being triggered to soon

hoogeveen opened the pull request:
The event is emitted before other fields are able to catch it.

hoogeveen committed:
Prevent event being triggered to soon

The event is emitted before other fields are able to catch it.

chielkunkels commented:
What exactly is it you're trying to solve? Often timeouts are not really the right solution. The event is supposed to fire as soon as things happen. Perhaps another event is needed or so to get the behaviour you require?

hoogeveen committed:
deployed a new version

hoogeveen committed:
added an hidden field

hoogeveen committed:
Added an hidden field in version 0.3.11

hoogeveen committed:
replaced innerText with TextContent

hoogeveen committed:
Keep wysiwyg's on trigger activation

hoogeveen committed:
Added type to classlist of field

hoogeveen committed:
Fix for wysiwyg initialization

hoogeveen committed:
Reverted wysiwyg changes

hoogeveen committed:
Fix for wysiwygs and version bump

amxfonseca committed:
Boolean should return false when not checked

amxfonseca committed:
Use the value provided by the field spec

Every field can specify their own getValue method. The onChange events should use that method to get the field value.

amxfonseca committed:
Added support for tips

amxfonseca committed:
Make the tooltip implementation simpler

Requires some extra CSS magic

amxfonseca committed:
Bumped version

amxfonseca committed:
Add option for a text field to subscribe others

This may break some features, revert if needed

amxfonseca committed:
Update package.json

hoogeveen committed:
Fix for firefox
